src/Query/Builder.php

```diff
@@ -266,7 +266,9 @@ public function toMql(): array
                     $aggregations = blank($this->aggregate['columns']) ? [] : $this->aggregate['columns'];
 
                     if (in_array('*', $aggregations) && $function == 'count') {
-                        return ['count' => [$wheres, []]];
+                        $options = $this->inheritConnectionOptions();
```

Comment:
If this is the fix for [PHPORM-65](https://jira.mongodb.org/browse/PHPORM-65), please make sure "PHPORM-65" appears in the final squash commit. I noted the commit includes a line explaining this fix,